### DISCUSSION TITLE
ScriptReference/Screen-resolutions

### DISCUSSION THREAD

**Discussion by docscomments** [Developer-Notes-Extension/Unity-Notes#299] [Notes]
# ScriptReference/Screen-resolutions

https://docs.unity3d.com/ScriptReference/Screen-resolutions.html

**Comment by joonjoonjoon** [comment]
NB: I have seen Screen.resolutions return an empty array on iOS.

**Reply by StinkySteak** [reply]
I stumble the the same issue with you but only in Android 5. Im using Unity 2021.3.21f1

**Comment by zhiganik** [comment]
Screen.resolutions returns the same HZ for each returned option

**Reply by vertxxyz** [reply]
This is untrue. It may be true in the editor, but I've only tested on one machine.
Example in a windows build:
![image](https://github.com/user-attachments/assets/98476872-42ae-4460-a547-8922028c27a0)

**Reply by StinkySteak** [reply]
and make sure to tell, if its a mobile or what kind of platform

**Reply by zhiganik** [reply]
it was the editor!
